app/questionnaire/placeholder_renderer.py

```diff
@@ -112,3 +126,36 @@ def render(
             set_pointer(dict_to_render, pointer, rendered_text)
 
         return dict_to_render
+
+    def check_transform(self, transform: MutableMapping[str, Any]) -> bool:
```

Comment:
Could we rename this specifically, i.e
`check_transform_needs_routing_path()` or something similar

And make it private

I find it makes the code easier to understand

Reply:
Updated and made private 👍